models/public/forward-tacotron/description/forward-tacotron.md

```diff
@@ -0,0 +1,132 @@
+5. Run provided script for conversion ForwardTacotron to onnx format
+```sh
+python3 forward_to_onnx.py --tts_weights checkpoints/ljspeech_tts.forward/fast_speech_step<iteration>K_weights.pyt
+```
+Notes:
+   1. Since ONNX doesn't support the build_index operation from PyTorch pipeline, the model is divided into two parts: `forward_tacotron_duration_prediction.onnx, forward_tacotron_regression.onnx`.
+   2. We stopped training of the Tacotron model in 183K iteration for alignment generation and stopped ForwardTacotron training in 290K iteration.
+
+## Composite model specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Source framework                | PyTorch*                                  |
+
+### Accuracy
+
+Subjective
+
+## forward-tacotron-duration-prediction model specification
+
+The forward-tacotron-duration-prediction model accepts preprocessed text (see text_to_sequence in [repository](https://github.com/as-ideas/ForwardTacotron/blob/78789c1aa845057bb2f799e702b1be76bf7defd0/utils/text/__init__.py)) and produces processed embeddings and
+duration in time for every processed embedding.
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| GOPs                            | 6.66                                      |
+| MParams                         | 13.81                                     |
+
+
+### Performance
+
+### Input
+
+Sequence, name: `input_seq`, shape: [1x241], format: [BxC]
+where:
+
+   - B - batch size
+   - C - number of symbols in sequence (letters or phonemes)
+
+### Output
+
+1. Duration for input symbols, name: `duration`, shape: [1, 241, 1], format [BxCxH]. Contains predicted duration for each of the symbol in sequence.
+   - B - batch size
+   - C - number of symbols in sequence (letters or phonemes)
+   - H - empty dimension.
+2. Processed embeddings, name: `embeddings`, shape: [1, 241, 512], format [BxCxH]. Contains processed embeddings for each symbol in sequence.
+   - B - batch size
+   - C - number of symbols in sequence (letters or phonemes)
+   - H - height of the intermediate feature map.
+
+## forward-tacotron-regression model specification
+
+The forward-tacotron-regression model accepts aligned by duration processed embeddings (for example: if duration is [2, 3] and processed embeddings is [[1, 2], [3, 4]], aligned embeddings is [[1, 2], [1, 2], [1,2], [3, 4], [3, 4]]) and produces mel-spectrogram.
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| GOPs                            | 4.91                                      |
+| MParams                         | 3.05                                      |
+
+### Performance
```

Comment:
we are removing empty sections from documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please remove empty section